src/test/kotlin/org/stellar/walletsdk/WalletTest.kt

```diff
@@ -1,25 +1,59 @@
 package org.stellar.walletsdk
 
 import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
 import kotlin.test.assertTrue
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
-import org.stellar.sdk.Network
 
-class WalletTest {
-  private val wallet = Wallet("https://horizon-testnet.stellar.org", Network.TESTNET.toString())
+internal class WalletTest {
+  private val wallet = Wallet(Constants.HORIZON_URL, Constants.NETWORK_PASSPHRASE)
 
-  @Test
-  fun `create() generates Stellar public and secret keys`() {
-    val accountKeys = wallet.create()
-    val publicKey = accountKeys.publicKey
-    val secretKey = accountKeys.secretKey
+  @Nested
+  @DisplayName("create")
+  inner class Create {
```

Comment:
I added nested tests with a display name to make it more clear in tests output which test comes from where.

![image](https://user-images.githubusercontent.com/7346473/185216473-a874372b-5352-4374-b291-a8aae51a763e.png)